starter_code/css/style.css

```diff
@@ -28,17 +29,129 @@ body {
     font-family: Helvetica, Arial, sans-serif;
 }
 
+.second{
+    background-color: #f0efef;
+    color: #606161;
+    font-family: 'open sans';
+    padding: 95px;
+}
 
-/******************************************
-/* LAYOUT
-/*******************************************/
+.third{
+    background-image: url(/images/testimonial_bg.jpg);
```

Comment:
Should be background-image: url(../images/testimonial_bg.jpg); to go up one directory